src/NzbDrone.Core/Indexers/Definitions/AnimeTorrents.cs

```diff
@@ -290,30 +327,30 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                 {
                     release.DownloadVolumeFactor = 0.5;
                 }
-                else
-                {
-                    release.DownloadVolumeFactor = 1;
-                }
 
                 var uLFactorImg = row.QuerySelector("img[alt*=\"x Multiplier Torrent\"]");
                 if (uLFactorImg != null)
                 {
                     release.UploadVolumeFactor = ParseUtil.CoerceDouble(uLFactorImg.GetAttribute("alt").Split('x')[0]);
                 }
-                else
-                {
-                    release.UploadVolumeFactor = 1;
-                }
-
-                qTitleLink.Remove();
 
-                //release.Description = row.QuerySelector("td:nth-of-type(2)").TextContent;
-                torrentInfos.Add(release);
+                releaseInfos.Add(release);
             }
 
-            return torrentInfos.ToArray();
+            return releaseInfos.ToArray();
         }
 
         public Action<IDictionary<string, string>, DateTime?> CookiesUpdater { get; set; }
     }
+
+    public class AnimeTorrentsSettings : UserPassTorrentBaseSettings
+    {
+        public AnimeTorrentsSettings()
+        {
+            DownloadableOnly = false;
+        }
+
+        [FieldDefinition(4, Label = "Downloadable Only", Type = FieldType.Checkbox, HelpText = "Search downloadable torrents only (enable this only if your account class is Newbie)")]
```

Comment:
can we detect/parse class after login and go off that? or is this best a user option?